From 180a897588c00c4c35d674ba16ad8a20bf5ac38b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 30 Jun 2017 01:11:53 +0200
Subject: notificationDaemon: Fix fallback icon names

The gtk-dialog-* names are old and non-standard, and haven't been
provided by the default icon theme for quite a while ...

https://bugzilla.gnome.org/show_bug.cgi?id=784245
---
 js/ui/notificationDaemon.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/js/ui/notificationDaemon.js b/js/ui/notificationDaemon.js
index df1a43d..6209625 100644
--- a/js/ui/notificationDaemon.js
+++ b/js/ui/notificationDaemon.js
@@ -128,10 +128,10 @@ const FdoNotificationDaemon = new Lang.Class({
         switch (hints.urgency) {
             case Urgency.LOW:
             case Urgency.NORMAL:
-                stockIcon = 'gtk-dialog-info';
+                stockIcon = 'dialog-information';
                 break;
             case Urgency.CRITICAL:
-                stockIcon = 'gtk-dialog-error';
+                stockIcon = 'dialog-error';
                 break;
         }
         return new Gio.ThemedIcon({ name: stockIcon });
-- 
cgit v0.12

